lib/plugins/aws/provider/awsProvider.js

```diff
@@ -341,6 +452,18 @@ class AwsProvider {
               required: ['localMountPath', 'arn'],
             },
             handler: { type: 'string' },
+            layers: { $ref: '#/definitions/awsLambdaLayers' },
+            memorySize: { $ref: '#/definitions/awsLambdaMemorySize' },
+            onError: { $ref: '#/definitions/awsSnsArn' },
```

Comment:
I'm not sure if it's worth secluding it into definitions, as it's not used elsewhere, and looks quite specific.

Also this definition is not clean, as it supports any ARN when expressed via CF function, and just SNS when ARN is passed as a string (it's fine to use it in context of `onError`, due to its implications, but as a reusable defunition, it feels dubious)